packages/components/psammead-most-read/src/item.jsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { node, oneOf, shape, string } from 'prop-types';
+import styled from 'styled-components';
+import { getFoolscap, getDoublePica } from '@bbc/gel-foundations/typography';
+import { C_EBON, C_POSTBOX } from '@bbc/psammead-styles/colours';
+import { scriptPropType } from '@bbc/gel-foundations/prop-types';
+import { GEL_SPACING } from '@bbc/gel-foundations/spacings';
+import { GEL_GROUP_3_SCREEN_WIDTH_MIN } from '@bbc/gel-foundations/breakpoints';
+import {
+  getSerifLight,
+  getSerifMedium,
+} from '@bbc/psammead-styles/font-styles';
+
+const MostReadWrapper = styled.div`
+  display: flex;
+
+  @media (max-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
+    margin-top: 0.25rem;
+  }
+
+  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
+    margin-top: 0.0625rem;
+  }
+`;
+
+const ItemWrapper = styled.div``;
+
+export const StyledCountSpan = styled.span`
+  ${({ script }) => script && getFoolscap(script)};
+  ${({ service }) => getSerifLight(service)}
+  color: ${C_POSTBOX};
+  margin: 0; /* Reset */
+  padding-bottom: ${GEL_SPACING};
+  display: inline-block;
+  min-width: 3rem;
+  
+  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
+    width: 4rem;
```

Comment:
We can have the width set at 3rem for all breakpoints. Then we can remove this media query and we shouldn't need the `min-width` value above either. This can simplify the CSS we have for this component.

Reply:
I don't believe this comment was addressed yet?